webapp/packages/plugin-settings-panel/src/locales/ru.ts

```diff
@@ -3,4 +3,7 @@ export default [
   ['plugin_settings_panel_search', 'Поиск настроек...'],
   ['plugin_settings_panel_no_settings', 'Настройки не найдены'],
   ['plugin_settings_panel_group_empty', 'Настройки недоступны'],
+  ['plugin_settings_panel_setting_reset', 'Сбросить'],
+  ['plugin_settings_panel_setting_reset_tooltip', 'Сбросить на значение по умолчанию'],
+  ['plugin_settings_panel_setting_set_in_scope', 'Настройка установлена в текущей области.'],
```

Comment:
области доступа?

Reply:
Not sure, it's not like the field of access, it's more like visibility scope or scope where it's applied, it may be "Настройка установлена в текущей области настроек" but...